src/public/confirm/styles.css

```
@@ -821,6 +821,16 @@ body {
   margin-top: 10px;
 }
 
+.modal-subtitle--reference {
+  margin-top: 8px;
+  font-size: 14px;
+  font-weight: 700;
+  letter-spacing: 0.2px;
+  color: var(--lavender);
+  opacity: 0.95;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)